controllers/toolchaincluster/healthchecker.go

```diff
@@ -68,7 +68,7 @@ func updateClusterStatuses(namespace string, cl client.Client) {
 			continue
 		}
 
-		clientSet, err := kubeclientset.NewForConfig(cachedCluster.Config)
+		clientSet, err := kubeclientset.NewForConfig(cachedCluster.Config.Config)
```

Comment:
Feels a bit odd to have Config within Config, maybe rename the inner one to RestConfig so that when you see `Config` you're not wondering which Config?

Reply:
this is a really good point :sweat_smile:  thanks ae1da04